tests/Unit/Http/Form/SignupFormTest.php

```diff
@@ -286,7 +286,58 @@ public function lastNameProvider(): array
     }
 
     /**
-     * Test that the Joind.in URL is being validated correctly
+     * Test that the joind.in username is being validated correctly
+     *
+     * @test
+     *
+     * @param string $joindin_username
+     * @param bool   $expectedResponse
+     * @dataProvider joindInUsernameProvider
+     */
+    public function joindInUsernameIsValidatedCorrectly($joindin_username, $expectedResponse)
+    {
+        $data['joindin_username'] = $joindin_username;
+
+        $form = new \OpenCFP\Http\Form\SignupForm($data, $this->purifier);
+        $form->sanitize();
+
+        $this->assertSame(
+            $expectedResponse,
+            $form->validateJoindInUsername(),
+            'Did not validate joind.in username as expected'
+        );
+    }
+
+    /**
+     * Data provider for joindInUsernameIsValidatedCorrectly
+     *
+     * @return array
+     */
```

Comment:
Perhaps we can remove the entire DocBlock?

Reply:
Yeah, not required for tests as far as I am concerned